UnitySetup/UnitySetup.psm1

```diff
@@ -2824,7 +2824,7 @@ function Update-UnityPackageManagerConfig {
         }
 
         Write-Verbose "Summary"
-        Write-Output $upmConfigs
+        Write-Verbose $upmConfigs
```

Comment:
I think we should either skip the summary or strip the tokens from the objects. Even in verbose it's not good practice.

Reply:
As a note - `Write-Output` is like adding it to the return value. It'll only get printed if the caller doesn't capture the output. That said, I agree that I think the author was trying to print this out for the summary not return it.